tasks/generate-sitemap.mjs

```diff
@@ -88,6 +88,13 @@ const createRoutePathList = (directory) => {
 
 const writeSitemap = async () => {
   const sitemapPath = './public/sitemap.xml';
+
+  // remove gen2 from being generated in the sitemap
+  const GEN2_ROUTE = "/gen2";
+  directory.children = directory.children.filter((node) => {
+    return node.route !== GEN2_ROUTE;
```

Comment:
Sorry if I'm dense, but does the router include the whole path? If so, then wouldn't we want to check if it starts with /gen2?